source/frontend/src/models/generate/GenerateAddress.ts

```diff
@@ -29,4 +30,29 @@ export class Api_GenerateAddress {
       .filter(a => !!a)
       .join('\n');
   }
+
+  public static fromIEditableOrgAddress(model: IEditableOrganizationAddress): Api_GenerateAddress {
+    let add = new Api_GenerateAddress(null);
+    add.line_1 = model.streetAddress1 ?? '';
+    add.line_2 = model.streetAddress2 ?? '';
+    add.line_3 = model.streetAddress3 ?? '';
+    add.city = model.municipality ?? '';
+    add.province = model.province ?? '';
+    add.postal = model.postal ?? '';
```

Comment:
this wouldn't be needed if we used Api_Organization concept instead of the IEditableOrganization...

Reply:
removed